msa/black-box-tests/src/test/java/org/thingsboard/server/msa/ui/base/AbstractDriverBaseTest.java

```diff
@@ -101,12 +100,13 @@ public void teardown() {
         driver.quit();
     }
 
-    public void openLocalhost() {
-        driver.get(getBaseUiUrl());
+    public String getJwtTokenFromLocalStorage() {
+        js = (JavascriptExecutor) driver;
+        return (String) js.executeScript("return window.localStorage.getItem('jwt_token');");
     }
 
-    public void openHomePage() {
-        driver.get(getBaseUiUrl() + "/home");
+    public void openLocalhost() {
```

Comment:
base url is something variable. not exactly localhost. I suggest setting another name for the method `openLocalhost() `